src/compiler/commandLineParser.ts

```diff
@@ -334,13 +335,42 @@ namespace ts {
         affectsSourceFile: true,
         affectsModuleResolution: true,
         affectsEmit: true,
+        affectsMultiFileEmitBuildInfo: true,
         paramType: Diagnostics.VERSION,
         showInSimplifiedHelpView: true,
         category: Diagnostics.Language_and_Environment,
         description: Diagnostics.Set_the_JavaScript_language_version_for_emitted_JavaScript_and_include_compatible_library_declarations,
         defaultValueDescription: ScriptTarget.ES3,
     };
 
+    /*@internal*/
+    export const moduleOptionDeclaration: CommandLineOptionOfCustomType = {

```

Comment:
Pulled this out so we can use moduleKind in impliedFormat of buildInfo to make it more readable when baselining